src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -544,7 +544,8 @@ type WebAppBuilder() =
                 Cors = config.Cors
                 Identity = config.Identity
                 ZipDeployPath = config.ZipDeployPath
-                AlwaysOn = state.AlwaysOn }
+                AlwaysOn = config.AlwaysOn 
```

Comment:
@isaacabraham  Part of this commit was also changing the state.AlwaysOn to config.AlwaysOn (with no particular intention but just copying the function application of this), but now I see that this would also solve [this issue](https://github.com/CompositionalIT/farmer/issues/532) as well